org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/CompletionTests18.java

```diff
@@ -5951,7 +5951,7 @@ public void testBug574823_completeOn_methodInvocationWithParams_inIfConidtion_in
 
 	String result = requestor.getResults();
     assertTrue(String.format("Result doesn't contain method forEach (%s)", result),
-    		result.contains("forEach[METHOD_REF]{forEach(), Ljava.lang.Iterable<Ljava.lang.String;>;, (Ljava.util.function.Consumer<-Ljava.lang.String;>;)V, null, null, forEach, (arg0), replace[149, 149], token[149, 149], 60}"));
+    		result.contains("forEach[METHOD_REF]{forEach(), Ljava.lang.Iterable<Ljava.lang.String;>;, (Ljava.util.function.Consumer<-Ljava.lang.String;>;)V, null, null, forEach, (arg0), replace[149, 149], token[149, 149], 55}"));
```

Comment:
@stephan-herrmann  Could this be due to poping out the `interestingEnclosings` ?

Reply:
> @stephan-herrmann Could this be due to poping out the `interestingEnclosings` ?

To assess the change in relevance I'd first check which ingredient to this number got lost.